tests/queries/0_stateless/02421_simple_queries_for_opentelemetry.sh

```diff
@@ -72,6 +85,7 @@ check_query_span "$query_id"
 query_id=$(${CLICKHOUSE_CLIENT} -q "select generateUUIDv4()");
 execute_query $query_id 'select * from opentelemetry_test format Null'
 check_query_span $query_id
+check_query_settings
```

Comment:
need to also update according `tests/queries/0_stateless/02421_simple_queries_for_opentelemetry.reference` file

Reply:
Update test script and reference file as per review comments